backend/app/templates/templates.go

```diff
@@ -0,0 +1,51 @@
+package templates
+
+import (
+	"io"
+	"io/ioutil"
+	"net/http"
+	"os"
+
+	log "github.com/go-pkgz/lgr"
+	"github.com/rakyll/statik/fs"
+)
+
+type File interface {
+	io.Closer
+	io.Reader
+	io.Seeker
+	Readdir(count int) ([]os.FileInfo, error)
+	Stat() (os.FileInfo, error)
+}
+type FileSystem interface {
+	ReadFile(name string) ([]byte, error)
+}
+
+type osFS struct{}
+
+var statikInstance http.FileSystem
+var isStatik bool
+
+func Init() FileSystem {
```

Comment:
We don't do that in Go. It's NewFileSystem, not Init, and it should return pointer.